Resources/config/routing.yml

```diff
@@ -91,13 +91,24 @@ canal_tp_mtt_line_timetable_list_defaults:
         seasonId: -?\d+
 
 canal_tp_mtt_line_timetable_list:
-    pattern: /line-timetable/networks/{externalNetworkId}/list/season/{seasonId}/line/{externalLineId}
-    defaults: { _controller: CanalTPMttBundle:LineTimetable:list, externalLineId : null }
+    pattern: /line-timetable/networks/{externalNetworkId}/line/{externalLineId}/list/season/{seasonId}
+    defaults: { _controller: CanalTPMttBundle:LineTimetable:list, seasonId : null }
```

Comment:
I corrected that in MttBridgeBundle. The route has changed to canal_tp_mtt_line_timetable_list_defaults